src/Common/StringUtils/StringUtils.h

```diff
@@ -138,6 +138,12 @@ inline bool isPrintableASCII(char c)
     return uc >= 32 && uc <= 126;   /// 127 is ASCII DEL.
 }
 
+inline bool isCSIFinalByte(char c)
```

Comment:
\+ `/// Control Sequence Introducer, https://en.wikipedia.org/wiki/ANSI_escape_code#CSIsection` for simple-minded people like me who don't know what "CSI" means